DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1

```diff
@@ -152,6 +156,12 @@ function Set-TargetResource
 
         # Add the IISSelfSignedCertModule native module to prevent self-signed certs being rejected.
         [boolean]$AcceptSelfSignedCertificates = $true,
+        
+       # Required Field when user want to enable DSC to use SQL server as backend DB
+       [boolean]$SqlProvider = $false,
```

Comment:
Why are we treating Sql provider as a special case here? Can we not give an option to use either ESENT/SQL/Access databases?

Reply:
in the previous logic - it assign the db provider based on the OS sku  - if it is win Blue or higher it use ese else for downlevel it uses Access
so there was no user input to choose which db provider to use 
so i did not want to change old behavior to support backward compatibility cases.

---
In reply to: [172346330](https://github.com/PowerShell/xPSDesiredStateConfiguration/pull/394#discussion_r172346330) [](ancestors = 172346330)